github/compliance/report.py

```diff
@@ -331,6 +333,224 @@ def milestone_for_max_days(
     )
 
 
+def _template_vars_cfg_element(
+    cfg_element_status_evaluation: gcm.CfgElementStatusEvaluation,
+) -> dict:
+
+    status = cfg_element_status_evaluation.status_report
+    evaluation = cfg_element_status_evaluation.evaluation_result
+
+    responsibles = '-'
+    if status.responsible:
+        # remove foremost "@" to prevent notification mails
+        responsibles = '<br/>'.join([
+            f'{r.name.removeprefix("@")} ({r.type.value})'
+            for r in status.responsible.responsibles
+        ])
+
+    policy_violations = ', '.join((violation.value for violation in evaluation.nonCompliantReasons))
+    summary = textwrap.dedent(f'''\
+        # Compliance Status Summary
+
+        |    |    |
+        | -- | -- |
+        | Cfg Storage | {status.element_storage} |
+        | Cfg Element | {status.element_name} |
+        | Cfg Type  | {status.element_type} |
+        | Policy Violation(s)  | {policy_violations} |
+        | Responsible(s)  | {responsibles} |
+    ''')
+
+    template_variables = {
+        'summary': summary,
+        'cfg_element_name': status.element_name,
+        'cfg_element_type':status.element_type,
+        'cfg_element_storage': status.element_storage,
+        'policy_violations': policy_violations,
+    }
+
+    return template_variables
+
+
+def create_or_update_github_issues_for_cfg(
```

Comment:
hmmm... this turns out to be more (partially redundant) LOCs than I hoped for :-/

-> that makes it more reasonable to do unify this w/ ResultGroupCollection, does it not?